runtime/src/accounts.rs

```diff
@@ -237,6 +238,45 @@ impl Accounts {
         })
     }
 
+    /// If feature `cap_transaction_accounts_data_size` is active, total accounts data a
+    /// transaction can load is limited to 64MiB to not break anyone in Mainnet today.
+    /// (It will be set by compute_bodget instruction in the future to more reasonable level).
+    fn get_requested_loaded_accounts_data_size_limit(
+        feature_set: &FeatureSet,
+    ) -> Option<NonZeroUsize> {
+        if feature_set.is_active(&feature_set::cap_transaction_accounts_data_size::id()) {
+            const REQUESTED_LOADED_ACCOUNTS_DATA_SIZE: usize = 64 * 1024 * 1024;
+            NonZeroUsize::new(REQUESTED_LOADED_ACCOUNTS_DATA_SIZE)
```

Comment:
It's a good point; it would be a big human error/fat-finger to define that const to zero, to guard against it,  I'd like `expect` more than unsafe `new_unchecked` call; and the suggested `bool.then` works well with `expect()` too.